exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/TransactionInvokerTest.java

```diff
@@ -16,6 +16,7 @@
 
 package com.exonum.binding.core.runtime;
 
+import static com.exonum.binding.core.runtime.TransactionInvokerTest.BasicService.TRANSACTION_ID;
```

Comment:
This import causes a compilation error on 8: 

```
[ERROR] /Users/travis/build/exonum/exonum-java-binding/exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/TransactionInvokerTest.java:[104,40] cannot find symbol

  symbol:   class Service

  location: class com.exonum.binding.core.runtime.TransactionInvokerTest
```

https://travis-ci.com/exonum/exonum-java-binding/jobs/270302809#L4823 (also locally with the most recent 8). Seems like a javac bug.

Reply:
The error message, as you see, is totally unrelated to the actual error.

Reply:
Yep, can reproduce locally with Java 8 on Mac.

Reply:
Thanks. Verbose compiler output didn't give any hints — I then reviewed the patch and thought it could be a candidate — and it was indeed: 
38b750f

Reply:
include paths, Service.class — everything it said it couldn't find was correct .